server/service/UserService.js

```diff
@@ -186,6 +186,13 @@ const userService = {
     if (!user) return {};
 
     logger.info("US: Will create a new password for " + user.email);
+
```

Comment:
Das würde doch bedeuten ich kann einen User für eine email Adresse anlegen die ich nicht besitze, oder?

Und wie kann er sich einloggen mit nicht aktivierten Account um das Passwort zu ändern?

Reply:
Hier geht es um das "Passwort vergessen".

Er bekommt ja zuerst einen Bestätigungslink an seine E-Mail-Adresse. Erst wenn er auf diesen Link geklickt hat, wird im ein neues Passwort zugesendet und jetzt in diesem Fall auch das Konto aktiviert.

Aber dazu musste er sich ja bereits vorher registrieren und die Mail-Adresse hat er ja durch den Bestätigungslink für das Passwort bestätigt.